README.md

```diff
@@ -20,7 +20,8 @@ Running the tests
 -----------------
 
 ```
-docker-compose run --rm ci venv/bin/pytest
+docker-compose -f docker-compose.yml -f docker-compose.ci.yml build
+docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm ci venv/bin/pytest
```

Comment:
Also having weird problems where the `ci` image is only rebuilt some of the time, or the `orcid-dummy` update is not pulled until a particular unrelated change.

Reply:
```
$ git diff
diff --git a/profiles/api/oauth2.py b/profiles/api/oauth2.py
index 34b19bd..242badd 100644
--- a/profiles/api/oauth2.py
+++ b/profiles/api/oauth2.py
@@ -32,6 +32,7 @@ def create_blueprint(orcid: Dict[str, str], clients: Clients, profiles: Profiles
     def _authorize() -> Response:
         if 'client_id' not in request.args:
             raise BadRequest('Invalid client_id')
+        random()
 
         try:
             client = clients.find(request.args.get('client_id'))
```
But `docker-compose -f docker-compose.yml -f docker-compose.ci.yml build` not noticing?
```
Building wsgi
Step 1/16 : ARG image_tag=latest
Step 2/16 : ARG python_version
Step 3/16 : FROM elifesciences/profiles_venv:${image_tag} as venv
 ---> 729b78a8b64c
Step 4/16 : FROM elifesciences/python_3.6:${python_version}
 ---> 3049b0227bd4
...
Step 11/16 : COPY --from=venv --chown=elife:elife ${PROJECT_FOLDER}/venv/ venv/
 ---> Using cache
 ---> ee34da3fce80
Step 12/16 : COPY --chown=elife:elife profiles/ profiles/
 ---> Using cache
 ---> 34e535302f3a
```

Reply:
The caching statement changing every time despite being a random addition?
```
Step 11/16 : COPY --from=venv --chown=elife:elife ${PROJECT_FOLDER}/venv/ venv/
 ---> Using cache
 ---> ee34da3fce80
Step 12/16 : COPY --chown=elife:elife profiles/ profiles/
 ---> Using cache
 ---> bf515e54e965
```

Reply:
Found it, it's `migrate` being built before `wsgi`

Reply:
We should extract and merge these fixes.